app/controllers/v1/base_controller.rb

```diff
@@ -53,4 +54,8 @@ def set_time_zone
     timezone = current_user.try(:timezone) || Time.zone.tzinfo.name
     Time.use_zone(timezone) { yield }
   end
+
+  def set_locale
+    I18n.locale = current_user.locale || I18n.default_locale
```

Comment:
If locale is validated on presence in user model, then there is no case when first part of right side will be "falsy".

Reply:
ah shit, yes